packages/@glimmer/interfaces/lib/components/component-manager.d.ts

```diff
@@ -21,6 +22,25 @@ export interface ComponentManager<
   getSelf(state: ComponentInstanceState): Reference;
   getDestroyable(state: ComponentInstanceState): Option<Destroyable>;
   getDebugName(state: ComponentDefinitionState): string;
+  getDebugRenderTreeName?(state: ComponentDefinitionState): string;
```

Comment:
We discussed this a bit separately, but I think we should remove this extra hook and handle the "while rendering" thing completely separately (e.g. where we have the `try`/`finally` during append/update VM's).